src/duplo_resource/jit.py

```diff
@@ -120,15 +120,17 @@ def web(self):
     }
   
   def __k8s_exec_credential(self, creds):
+    c = {
```

Comment:
can we give a meaningfull variable name here for `c`?

Reply:
I suppose I could. I am a fan of single letter vars when the docs are good.